docs/source/compatibility.rst

```diff
@@ -13,59 +15,45 @@ Note that this document may contain ambiguities on the level of supported compat
 Targeted Versions
 -----------------
 
-This policy is applied to Chainer of versions v1.5.1 and higher.
+**This policy is applied to Chainer v2.0.0 and higher.**
 Note that this policy is not applied to Chainer of lower versions.
+For older versions of Chainer, see `the old version of API Compatiblity Policy <http://docs.chainer.org/en/v1.24.0/compatibility.html>`_.
 
 
-Versioning and Backward Compatibilities
----------------------------------------
-
-The updates of Chainer are classified into three levels: major, minor, and revision.
-These types have distinct levels of backward compatibilities.
-
-- **Major update** contains disruptive changes that break the backward compatibility.
-- **Minor update** contains addition and extension to the APIs keeping the supported backward compatibility.
-- **Revision update** contains improvements on the API implementations without changing any API specifications.
-
-Note that we do not support full backward compatibility, which is almost infeasible for Python-based APIs, since there is no way to completely hide the implementation details.
-
+Versioning and Backward Compatibility
+-------------------------------------
 
-Processes to Break Backward Compatibilities
--------------------------------------------
+The versioning of Chainer follows the `PEP 440 <https://www.python.org/dev/peps/pep-0440/>`_ and a part of `Semantic versioning <http://semver.org/>`_.
+See :ref:`contrib` for details of versioning.
 
-Deprecation, Dropping, and Its Preparation
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+The backward compatibility is kept for **revision updates**, which are applied to the latest stable version.
```

Comment:
I think we should remove "latest" because that could mistakenly make users think that there are multiple stable versions, which is not true.